### PR TITLE
[Calendar] Fix Calendar.getEventsAsync and remove Calendar.DEFAULT

### LINKED ISSUE #4760: [Calendar][Android] Expo crashes when Calendar.getEventsAsync is called on Android
### Environment

Expo CLI 2.20.10 environment info:
    System:
      OS: macOS 10.14.5
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 11.3.0 - /usr/local/bin/node
      Yarn: 1.15.2 - /usr/local/bin/yarn
      npm: 6.9.0 - /usr/local/bin/npm
    IDEs:
      Android Studio: 3.4 AI-183.5429.30.34.5452501
      Xcode: 10.2.1/10E1001 - /usr/bin/xcodebuild
    npmPackages:
      expo: ^33.0.0 => 33.0.7
      react: 16.8.3 => 16.8.3
      react-native: https://github.com/expo/react-native/archive/sdk-33.0.0.tar.gz => 0.59.8
      react-navigation: ^3.0.9 => 3.0.9
    npmGlobalPackages:
      expo-cli: 2.20.10

Bug happens only in Android (all versions I tested) when testing my app on both emulators and real devices

### Steps to Reproduce

I tried to call `Calendar.getEventsAsync()` on `Calendar.DEFAULT` between two Dates instances.

### Actual Behavior

Expo app crashes

### Reproducible Demo

https://snack.expo.io/SJKlTKXxH

### PR DESCRIPTION
**Pull request by lukmccall**
# Why

Resolves #4760.

# How

Wrapping calendarId with ''.

# Test Plan

If calendarId is sqlite keyword it will not longer crash app.
You can check it https://snack.expo.io/@lukaszkosmaty/healthy-crackers.

### CODE REVIEW

File: packages/expo-calendar/ios/EXCalendar/EXCalendar.m [hunk 1]
```diff
@@ -285,8 +283,15 @@ - (EKEventStore *)eventStore
...
         return;
-      }
+      } 
+      
+      calendarEvent = [EKEvent eventWithEventStore:self.eventStore];
       calendarEvent.calendar = calendar;
+    } else {
+        reject(@"E_INVALID_CALENDAR_ID",
+             @"CalendarId is required.",
+             nil);
```
Comment (bbarthec) [inline packages/expo-calendar/ios/EXCalendar/EXCalendar.m]: Two things:
1) proper indentation
2.a) in general if method invocation isn't too long, we keep it as oneliner (`reject(@"E_INVALID_CALENDAR_ID", @"CalendarId is required.", nil);`)
2.b) another thing is to keep an eye on parameters alignment when method invocation is multiliner - keep all arguments aligned to the first one
```obj-c
       |
reject(@"E_INVALID_CALENDAR_ID",
       @"CalendarId is required.",
       nil);
       |
       ^
       aligning line 
```

File: packages/expo-calendar/src/Calendar.ts [hunk 2]
```diff
@@ -243,7 +243,7 @@ export async function getEventAsync(
   }
 }
 
-export async function createEventAsync(calendarId: string, details: OptionalKeys<Event> = {}): Promise<string> {
+export async function createEventAsync(calendarId: string, { id, ...details }: OptionalKeys<Event> = {}): Promise<string> {
```
Comment (bbarthec) [inline packages/expo-calendar/src/Calendar.ts]: I have a quest for you 😂 Please, in separate commit, remove type `OptionalKeys` and replace it with TypeScript default `Partial` built-in type 😜

File: packages/expo-calendar/src/Calendar.ts [hunk 3]
```diff
@@ -377,7 +377,7 @@ export async function deleteAttendeeAsync(id: string): Promise<void> {
 } // Android
 
 export async function getRemindersAsync(
-  calendarIds: string[],
+  calendarIds: (string | null)[],
```
Comment (bbarthec) [inline packages/expo-calendar/src/Calendar.ts]: Interesting way, but convention is to use `Array<firstTypeInUnionType | secondTypeInUnionType | ... and so on>`

File: packages/expo-calendar/src/Calendar.ts [hunk 4]
```diff
@@ -419,21 +419,16 @@ export async function getReminderAsync(id: string): Promise<Reminder> {
 } // iOS
 
 export async function createReminderAsync(
-  calendarId: string,
-  details: Reminder = {}
+  calendarId: string | null,
+  {id, ...details}: Reminder = {}
```
Comment (bbarthec) [inline packages/expo-calendar/src/Calendar.ts]: proper indentation is:
`{ id, ...details }: Reminder = {}`

**Review by bbarthec — CHANGES_REQUESTED** [review]
Nice job 💯 👍 💪 Please refer to my comments and add entry in [CHANGELOG.md](https://github.com/expo/expo/blob/master/CHANGELOG.md)

**Review by tsapeta — APPROVED** [review]
Nice one, good work! 💪 Let @bbarthec do the final review and feel free to merge once he approves 😉

**Review by bbarthec — APPROVED** [review]

**Pull request merged**